Jenkinsfile.daily

```diff
@@ -0,0 +1,65 @@
+properties([
+    pipelineTriggers([cron('H H(1-3) * * *')]),
+    buildDiscarder(logRotator(numToKeepStr: '30', artifactNumToKeepStr: '30')),
```

Comment:
Is it possible to fetch this from a single place? I assume that will also be needed by other Jenkinsfiles?

Reply:
While there are shared libraries I don't think it makes sense here as those settings are specific to that build job.